### PR TITLE
DFC-13315-Updated design changes needed for PSF

### PR DESCRIPTION
**Pull request by vallabha24**

### CODE REVIEW

**Review by jojestesfa — APPROVED** [review]

**Review by Gurnaik-Randhawa — DISMISSED** [review]

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml [hunk 1]
```diff
@@ -2,59 +2,73 @@
...
-                        </ol>
-                </div>
-              @Html.HiddenFor(m => Model.Section.Name)
-              @Html.HiddenFor(m => Model.Section.SectionDataType)
-              @Html.HiddenFor(m => Model.OptionsSelected)
-              @Html.HiddenFor(m => Model.Section.PageNumber)
-              @Html.HiddenFor(m => Model.Section.SingleSelectOnly)
+        <form action="@Model.Section.PreviousPageUrl" method="post" id="backForm">
+            <div id="BackLink" class="govuk-breadcrumbs">
+                <a href="@Model.Section.PreviousPageUrl" class="govuk-back-link" id="filter-home">@(Model.Section.PageNumber == 1 ? "Home" : "Back")</a>
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml:7]: It will have to be a button, as the form needs to be submitted to preserve state.

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml [hunk 2]
```diff
@@ -2,59 +2,73 @@
...
+            @Html.HiddenFor(m => Model.Section.PageNumber)
+            @Html.HiddenFor(m => Model.Section.SingleSelectOnly)
         </form>
         <form action="@Model.Section.NextPageUrl" method="post" id="continueForm">
             <div class="active">
-                <h4>Step @Model.Section.PageNumber of @Model.Section.TotalNumberOfPages@if (@Model.Section.PageNumber > 1){<span> you have @Model.NumberOfMatches career matches</span>}</h4>
-                <h2>@Html.DisplayFor(model => Model.Section.Name)</h2>
-                @Html.Raw(Model.Section.Description)      
+                <span class="govuk-caption-xl">Step @Model.Section.PageNumber of @Model.Section.TotalNumberOfPages</span>
+                @*@if (@Model.Section.PageNumber > 1)
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml:19]: Why is this removed? has this been moved to to correct HTML section?

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml [hunk 3]
```diff
@@ -2,59 +2,73 @@
...
+                                        <label class="govuk-label govuk-radios__label" for="Section_@jj.ToString()">@Model.Section.Options[jj].Name<span>@Model.Section.Options[jj].Description</span></label>
+                                    </div>
+                                }
+                                else if (Model.Section.SectionDataType == "TrainingRoute")
+                                {
+                                    <div class="filter-option govuk-radios__item">
+                                        @Html.RadioButtonFor(m => Model.Section.SingleSelectedValue, Model.Section.Options[jj].OptionKey, new { id = $"Section_{jj.ToString()}", @class = "govuk-radios__input" })
+                                        <label class="govuk-label govuk-radios__label" for="Section_@jj.ToString()">@Model.Section.Options[jj].Name<span>@Model.Section.Options[jj].Description</span></label>
+                                    </div>
+                                }
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml:52]: What is this special case used for? looks very similar to SingleSelect as it should be

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml [hunk 4]
```diff
@@ -2,59 +2,73 @@
...
-                    }
+                    </div>
                 </div>
-                <button type="submit" class="button filter-next" id="filter-continue">Continue</button>
-            </div>
-            </form>
+
+
+                <button type="submit" class="govuk-button filter-next" id="filter-continue">Continue</button>
+                <a href="#" class="govuk-link govuk-!-margin-left-5">Skip this step</a>
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml:70]: Don't think # is correct for "skip", this needs to be correctly coded

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PsfSearch/SearchResult.cshtml [hunk 5]
```diff
@@ -4,33 +4,38 @@
...
 
-            <div id="results">
-                <div class="results-block">
-                    <div class="inner-block">
+            <div class="govuk-warning-text">
+                <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                <strong class="govuk-warning-text__text">
+                    <span class="govuk-warning-text__assistive">Warning</span>
+                    If the job roles you have been matched to are COVID impacted, you should
+                    <a href="#" class="govuk-link">contact our advisers</a> to consider your options and next steps.
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PsfSearch/SearchResult.cshtml:23]: In correct anchor link #

**Review by Muthuramana — CHANGES_REQUESTED** [review]

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml (hunk 3 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml:52]: This was created for Trainingroute which is used for Covid-19 options which are Radio buttons. There was another SingleSelect  section but I kept them separate so that if we need to style the TrainingRoute on its own it will be easier. I changed that btw

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml [hunk 6]
```diff
@@ -2,59 +2,73 @@
...
                 @Html.HiddenFor(m => Model.Section.SingleSelectOnly)
 
-                <div class="filter-list">
-                    @if (Model.Section.Options != null && Model.Section.Options.Any())
-                    {
-                        for (int jj = 0; jj < Model.Section.Options.Count; jj++)
+                <div class="govuk-form-group">
+                    <fieldset class="govuk-fieldset" aria-describedby="qualifications-hint">
+                        <div class="govuk-hint"> Select all that apply. </div>
+                    </fieldset>
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml:33]: Shouldn't this field set enclose all of the input elements? it should be closing just before the end of form tag

**Review by Muthuramana — CHANGES_REQUESTED** [review]

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml (hunk 1 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml:7]: Changed to button for now and Joe was looking into the style change for it and will be added in the next release

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml (hunk 4 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml:70]: Skip this step was removed as agreed. This cant be just a URL and it must be a submit button which isn't possible when the Continue is also a submit button

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml (hunk 6 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml:33]: This was updated

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml (hunk 2 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PreSearchFilters/Index.cshtml]: This was not needed as that part was not MVP and cant be done by 12th Nov. I removed that completely anyway instead of commenting it for now

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PsfSearch/SearchResult.cshtml (hunk 5 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/PsfSearch/SearchResult.cshtml]: Updated the anchor link to contact-us page

**Review by Muthuramana — APPROVED** [review]
